Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml

```
@@ -116,6 +116,9 @@ script:
       name: source
     - description: Additional alert note.
       name: note
+    - name: details
+      description: Comma-separated key-value pairs to use as custom properties of the alert. JSON format is also supported when used within an automation.
```

```suggestion
      description: Comma-separated key=value pairs to use as custom properties of the alert. JSON format is also supported when used within an automation.
```
Maybe you should also add an example